lib/Backend/Sym.cpp

```diff
@@ -1090,9 +1091,14 @@ Sym::Dump(IRDumpFlags flags, const ValueType valueType)
                             uint index = stackSym->GetByteCodeRegSlot() - stackSym->GetByteCodeFunc()->GetJITFunctionBody()->GetConstCount();
                             Js::PropertyId propertyId = functionBody->GetPropertyIdOnRegSlotsContainer()->propertyIdsForRegSlots[index];
                             Output::Print(_u("(%s)"), stackSym->GetByteCodeFunc()->GetInProcThreadContext()->GetPropertyRecord(propertyId)->GetBuffer());
+                            printed = true;
                         }
                     }
                 }
+                if (!printed)
+                {
+                    Output::Print(_u("{%d}"), stackSym->GetByteCodeRegSlot());
```

Comment:
>{%d} [](start = 38, length = 4)

Is it necessary to introduce the new format `{%d}` which doesn't look like a general usage?